src/AppBundle/Controller/Admin/VoteSettingsController.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+namespace AppBundle\Controller\Admin;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use AppBundle\Entity\VoteSettings;
+use AppBundle\Form\VoteSettingsType;
+
+/**
+ * VoteSettings controller.
+ *
+ * @Route("/admin/vote_settings")
+ */
+class VoteSettingsController extends Controller
+{
+    /**
+     * Lists all VoteSettings entities.
+     *
+     * @Route("/", name="vote_settings")
+     * @Method("GET")
+     * @Template()
+     */
+    public function indexAction()
+    {
+        $em = $this->getDoctrine()->getManager();
+
+        $entities = $em->getRepository('AppBundle:VoteSettings')->findAll();
+
+        return array(
+            'entities' => $entities,
+        );
+    }
+    /**
+     * Creates a new VoteSettings entity.
+     *
+     * @Route("/", name="vote_settings_create")
+     * @Method("POST")
```

Comment:
look this  - 
If we have vector about development like Domain-Driven Design, DDD - then we crate one function in domain and only this function create new instans. This is two function small example 
about responsibility layer, for create new instans responsibility severally function